draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3343,7 +3399,7 @@ RESPONSE:
 	</figure>
 	</section>
 
-	<section title="Invalidate" anchor="sect-6.1.2"><t>
+	<section title="Invalidate" anchor="examples-invalidate2"><t>
```

Comment:
Any reason to call it "examples-invalidate2" when there doesn't seem to be an "examples-invalidate"?

Reply:
typo, idnits didn't find it because it is not referenced anywhere